target/linux/ath79/dts/ar9331_embeddedwireless_dorin-mobile-gw.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+ #include "ar9331_embeddedwireless_dorin.dts"
+
+ / {
+ 	model = "Embedded Wireless Mobile Gateway";
+ 	compatible = "embeddedwireless,dorin-mobile-gw", "qca,ar9331";
+ };
+
+ &wmac {
+ 	status = "disabled";
+ };
```

Comment:
The indention looks strange, I think everything is indented by one.

Reply:
Thanks, I'll reformat and send again.